src/extension.ts

```diff
@@ -23,11 +23,13 @@ import { WorkspaceListener } from "./utils/listener/workspaceListener";
 import { DocAndFeedbackView } from "./views/docsAndFeedbackView/docAndFeedbackView";
 import { messages } from "./utils/common/messages";
 import { commands } from "./utils/common/commands";
+import { VorpalCommand } from "./commands/vorpalCommand";
 
 export async function activate(context: vscode.ExtensionContext) {
   // Create logs channel and make it visible
   const output = vscode.window.createOutputChannel(constants.extensionFullName);
   const logs = new Logs(output);
+  logs.show();
```

Comment:
If i'm not wrong, we add this logs.show() in other place, there was a story created to remove it because it's annoying some of the customers: https://checkmarx.atlassian.net/browse/AST-37522

Reply:
It's interesting that it is not merged into my PR...
I will fix it